### ISSUE TITLE
Training loss for critic not converge. 

### ISSUE THREAD

**Issue by akira-l** [daerduoCarey/where2act#14]
I follow the instrcution in the repo, collecting data and then training the critic at first. 
But I met problem at the first model training as critic. I found the loss is hard to converge as follow: 
![image](https://user-images.githubusercontent.com/9478838/210672810-9fc6727f-abcd-4884-ad87-a40d95d76703.png)
Above loss figure is in 34 epochs. 

Do I need to wait for converage ? I see the provided checkpoints are about 100 epochs. 
So could you pls provide a log for critic? I'd like to known what the usual condition of converage for this loss function. 
BTW, another solution is just wait for about 100 epochs to reach the similar training steps to your opened checkpoints. Is that the convergence not necessary in this case?
